### ISSUE TITLE
No available provider "heroku" for terraform 0.12

### ISSUE THREAD

**Issue by vukanac** [terraform-providers/terraform-provider-heroku#200]
Hi there,

On the clean machine (mac os x), installed `brew install terraform`.

```
$ terraform -v
Terraform v0.12.0
```

`main.tf`:
```hcl
provider "heroku" {
  version = "~> 1.9"
}
```

### Received error
`No available provider "heroku" plugins are compatible with this Terraform version.`

### Steps to Reproduce
1. `terraform init`

**Comment by mars** [comment]
I'm surprised Hashicorp did not make the new version available of official providers like this one. We'll look into making this work with Terraform 0.12.

**Comment by ryansdwilson** [comment]
Excited to see your PR get merged! Looking forward to using Terraform 0.12 w/ heroku.

**Comment by mars** [comment]
Working to resolve this over here #179

**Comment by mars** [comment]
Terraform Provider Heroku v2.0.0 has been released with support for Terraform 0.12!

**Comment by vukanac** [comment]
main.tf:

```
provider "heroku" {
  version = "~> 2.0"
}
```
